evap/grades/templates/grades_semester_base.html

```diff
@@ -0,0 +1,23 @@
+{% extends "grades_base.html" %}
+
+{% load i18n %}
+
+{% block subtitle %}
+    {{ block.super }}
+    {% if semester %}
+        {% if disable_breadcrumb_semester %}
+            <li>{{ semester.name }}</li>
+        {% else %}
+            <li><a href="{% url "grades:semester_view" semester.id %}">{{ semester.name }}</a></li>
+        {% endif %}
+    {% endif %}
+{% endblock %}
+
+{% block title %}
+    {% if semester %}
+        {{ semester.name }} -
+    {% else %}
+        {% trans "Semesters" %} -
+    {% endif %}
```

Comment:
the title is used in the title bar of the bowser. here we add the semester name (or the string `Semesters` if we don't have a semester at hand, e.g. when we are in a semester overview).